dh.c

```diff
@@ -152,11 +155,48 @@ choose_dh(int min, int wantbits, int max)
 	int best, bestcount, which, linenum;
 	struct dhgroup dhg;
 
+#ifndef WINDOWS
 	if ((f = fopen(_PATH_DH_MODULI, "r")) == NULL) {
 		logit("WARNING: could not open %s (%s), using fixed modulus",
 		    _PATH_DH_MODULI, strerror(errno));
 		return (dh_new_group_fallback(max));
 	}
+#else
+	/* First check the moduli file in the %programdata%\ssh\ directory.
+	 * If not then search for the moduli file in the current executable directory. This file will be updated in new OpenSSH releases.
```

Comment:
do we have a script that generates the moduli file (for every release) ? will be good to include that too.

Reply:
We don't have script to generate this file. It will take multiple days to generate a higher prime number like 8192 bits.
Having a script doesn't help here.